tests/scripts/thread-cert/Cert_7_1_03_BorderRouterAsLeader.py

```diff
@@ -27,18 +27,21 @@
 #  POSSIBILITY OF SUCH DAMAGE.
 #
 
+from binascii import hexlify
 import time
 import unittest
-
+import mle
 import config
+import command
+from command import CheckType
```

Comment:
Maybe sort out the imports? I think it's a little bit weird, `import command` with the next line `from command import CheckType`. And I think we should unify the usage of imports, when to use `import` and when to use `from ... import ...`. Currently, in this file, I think we can sort the imports at least.